theme.json

```diff
@@ -2,7 +2,50 @@
 	"settings": {
 		"appearanceTools": true,
 		"color": {
-			"defaultPalette": false,
+			"defaultDuotone": false,
+      "defaultPalette": false,
+			"duotone": [
+				{
+					"colors": [
+						"#111111",
+						"#ffffff"
+					],
+					"slug": "black-and-white",
+					"name": "Black and white"
+				},
+				{
+					"colors": [
+						"#111111",
+						"#C2A990"
+					],
+					"slug": "duotone-1",
+					"name": "Duotone 1"
```

Comment:
shall we do it for all of them? 🙈

Reply:
No, I don’t think “Accent 1 and Contrast” is very helpful. You won’t remember that; just the applied tones.

Reply:
And I think it’s better to keep slugs generic, so they’re more interoperable.

Reply:
I think that would be the way to go if duotone presets were actually dynamic but since they are not, a more descriptive name would be more useful for the user (black and red, rather than accent 1 and contrast)

Reply:
For a user with visual impairments, a descriptive color term is significantly more useful than an abstraction.

Reply:
@richtabor Not really.  It doesn't give any context unless you have visual recent, in my opinion.

Reply:
Nobody is actively working on that feature that we know of.

If we call it "red" and change that color in the global style variations, then we only need to include the duotone in the variation file too.

Reply:
Made the change — thanks for chiming in!